front/lib/resources/data_source_resource.ts

```diff
@@ -343,7 +317,7 @@ export class DataSourceResource extends ResourceWithVault<DataSource> {
     await DataSourceViewModel.destroy({
       where: {
         workspaceId: auth.getNonNullableWorkspace().id,
-        dataSourceId: this.id,
+        dataSourceId: this.id.toString(),
```

Comment:
To fix: https://app.datadoghq.eu/logs?query=%22Unhandled%20activity%20error%22%20service%3Afront-worker%20env%3Aprod%20&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&event=AgAAAZHmXe-w0Iv8xAAAAAAAAAAYAAAAAEFaSG1YZjdIQUFBdDdRTzNiS04xQVFCaAAAACQAAAAAMDE5MWU2NWUtMGQ4Zi00OTM1LWE4NTMtZGU4YzFjOTI5Mzdl&fromUser=true&messageDisplay=inline&refresh_mode=sliding&source=monitor_notif&storage=hot&stream_sort=desc&viz=stream&from_ts=1726145911000&to_ts=1726146811000&live=false